crates/chain/src/local_chain.rs

```diff
@@ -545,31 +545,50 @@ impl std::error::Error for ApplyHeaderError {}
 
 /// Applies `update_tip` onto `original_tip`.
 ///
-/// On success, a tuple is returned `(changeset, can_replace)`. If `can_replace` is true, then the
-/// `update_tip` can replace the `original_tip`.
+/// On success, a tuple is returned `([`CheckPoint`], [`ChangeSet`])`.
```

Comment:
Maybe remove the outer set of backticks
```suggestion
/// On success, a tuple is returned ([`CheckPoint`], [`ChangeSet`]).
```